### DISCUSSION TITLE
How to set timeout workflow for send HTTP

### DISCUSSION THREAD

**Discussion by pepea23** [elsa-workflows/elsa-core#3091] [Q&A]
How to set timeout workflow for send HTTP request. need help 🥺

![image](https://user-images.githubusercontent.com/32633442/170056517-e945d940-b7a2-406f-a328-23f74c53270a.png)

**Comment by sfmskywalker** [comment]
Hi @pepea23 

You can configure the HTTP client that is used by the Send HTTP Request activity from your Startup/Program. For example:

```csharp
services.AddElsa(elsa => elsa.AddHttpActivities(configureHttpClient: builder => builder.ConfigureHttpClient(httpClient => httpClient.Timeout = TimeSpan.FromMinutes(10))));
```

This will configure the HTTP client with a timeout of 10 minutes.

**Reply by pepea23** [reply]
I have use docker for run.. so do you have  guideline configure ? 🥺
